spec/requests/unwithdraw_spec.rb

```diff
@@ -1,52 +1,71 @@
 # frozen_string_literal: true
 
 RSpec.describe "Unwithdraw" do
-  let(:edition) { create(:edition, :withdrawn, :political, :past_government) }
+  let(:managing_editor) { create(:user, managing_editor: true) }
```

Comment:
I'd be tempted to have a `let(:user_permissions) { { managing_editor: true } }` at the top of the file, and change this line to:

```suggestion
  let(:managing_editor) { create(:user, user_permissions) }
```

That way we can nicely set a `let(:user_permissions) { { managing_editor: true, manage_live_history_mode: true } }` on line 22 and make that test consistent with the rest of these.

Reply:
That does slip into a world where we need every change of user to be in a different context. I'll let Pete see how he feels about that but does seem to add some extra need for context blocks and more variables outside of test.

Reply:
I'd have to agree with Kevin on this one, nice suggestion though!